test/Chainweb/Test/Pact/PactMultiChainTest.hs

```diff
@@ -237,6 +239,27 @@ chainweb213Test = do
         , "(create-table tbl)"
         ]
 
+pactLocalMaxGasLimitTest :: PactTestM ()
+pactLocalMaxGasLimitTest = do
+  runToHeight 53
```

Comment:
Can you include a comment for why this is 53?

Reply:
Removed as it doesn't make any difference here. Was a leftover from copying the test.